CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ Added wording to Code of Conduct to include in-person events.
 
 August 16, 2016
 
-    Two Sections Added:  
+```
```

Comment:
i'm going to ignore issues like this, it's the wrong formatting, but at least it uses the right syntax.